.travis.yml

```diff
@@ -23,5 +23,4 @@ addons:
 script:
   - ./.gofmt.sh
   - make
-  - bazel build //:go_default_test
```

Comment:
why are you removing this from travis?

Reply:
Ah, it's been essentiall moved into tests/structure_test.sh so this was redundant. the bazel build //:structure_test_image in there does this build as well, since it's a dependency.

Reply:
ah ok, makes sense.